dbms/src/Storages/Page/V3/BlobStore.cpp

```diff
@@ -1353,10 +1352,9 @@ BlobStatPtr BlobStore::BlobStats::createBigPageStatNotChecking(BlobFileId blob_f
     return stat;
 }
 
-void BlobStore::BlobStats::eraseStat(const BlobStatPtr && stat, const std::lock_guard<std::mutex> &)
+void BlobStore::BlobStats::eraseStat(const BlobStatPtr && stat, const std::lock_guard<std::mutex> & lock)
 {
-    PageFileIdAndLevel id_lvl{stat->id, 0};
-    stats_map[delegator->getPageFilePath(id_lvl)].remove(stat);
```

Comment:
I wonder why doesn't this method return correct path

Reply:
This is also correct when run tiflash as a service. But for `BlobStoreStatsTest` in gtest_blob_store.cpp, it doesn's create any BlobFile, so it doesn't have any info in delegator, and the unit test cannot run.